mkdocs/tests/config/config_options_tests.py

```diff
@@ -494,6 +494,50 @@ def test_dir_is_config_dir_fails(self):
         self.assertEqual(len(warns), 0)
 
 
+class ListOfPathsTest(unittest.TestCase):
+
+    def test_valid_path(self):
+        paths = [os.path.dirname(__file__)]
+        option = config_options.ListOfPaths()
+        option.validate(paths)
+
+    def test_missing_path(self):
+        paths = [os.path.join("doesnt", "exist", "i", "hope")]
+        option = config_options.ListOfPaths()
+        self.assertRaises(config_options.ValidationError, option.validate, paths)
```

Comment:
```suggestion
        with self.assertRaises(config_options.ValidationError):
            option.validate(paths)
```
as per https://github.com/mkdocs/mkdocs/pull/2653